std/math.d

```diff
@@ -3927,6 +3927,91 @@ float floor(float x) @trusted pure nothrow @nogc
     assert(isNaN(floor(float.init)));
 }
 
+/**
+ * Round `val` to a multiple of `unit`. `rfunc` specifies the rounding
+ * function to use; by default this is `rint`, which uses the current
+ * rounding mode.
+ */
+real quantize(alias rfunc = rint)(real val, real unit)
```

Comment:
> Don't we also want `double` and `float`?

I made it `real`-only because that's all that `rint()`, `round()`, and `nearbyint()` really support right now. However, I see now that `floor()` and `ceil()` have proper `double` and `float` implementations, so I will update `quantize()` accordingly.